Sources/DoctorPretty/Enclosed.swift

```diff
@@ -14,9 +14,9 @@ extension BidirectionalCollection where Iterator.Element == Doc, IndexDistance =
     /// Intersperses punctuation inside docs
     public func punctuate(with punctuation: Doc) -> [Doc] {
         if let d = first {
-            return [d] + self.dropFirst().reduce([Doc]()) { t in
-                let (acc, d2) = t
-                return acc <> [punctuation, d2]
+            return [d] + self.dropLast().reduce(into: [Doc]()) { acc, d2 in
```

Comment:
theoretically `reduce(into:)` will be more performant

Reply:
accidental dropFirst -> dropLast

Reply:
oh duh! dangit